src/types/markers.js

```diff
@@ -386,6 +386,8 @@ export type DOMEventMarkerPayload = {|
   eventType: string,
   phase: 0 | 1 | 2 | 3,
   cause?: CauseBacktrace,
+  docShellId?: string,
+  docshellHistoryId?: number,
```

Comment:
Not sure if it's fine to simply add these to the type, but evidently, tracing markers can (do?) have this docshell data and these were needed to make flow accept my changes to the test.

Reply:
And this should also likely be in other places too. Maybe we need a `BasicPayload` type that we could reuse. Anyway this is out of the scope of this issue :)